generators/react/templates/base/src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
Un fichier avec juste une ligne commenté sans explication c'est un peu aride :(

Reply:
It's a typescript [directive](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html) that is generated like this by `create-vite`.

It's a technicality of how typescript work and this file should never be touched, not sure if it's worth adding a comment to explain it.